yazi-adapter/src/image.rs

```diff
@@ -40,7 +40,7 @@ impl Image {
 				img.as_bytes(),
 				img.width(),
 				img.height(),
-				img.color().into(),
```

Comment:
This is intentional because the `image` crate v0.25 changed the type, and Yazi downgraded quickly after an upgrade since `image` v0.25.1 [contains a bug](https://github.com/etemesi254/zune-image/issues/184) that causes the program to crash. 

However, once they fix it, we won't need to change this part when we upgrade again.